docs/Prefill-decoding_Disaggregation.md

```diff
@@ -42,8 +42,33 @@ Currently P-D disaggregation is an experimental feature, mainly to demonstrate t
 We are actively working on these items. Stay tuned :)
 
 ## How to use
-Llumnix uses two simple arguments to enable prefill-decoding disaggregation in the current version. 
-- `--enable-pd-disagg True` is used to enable prefill-decoding disaggregation.
-- `--num-available-dispatch-instances` is used to configure the initial number of prefill instances. 
 
-Note that one should make sure that `num-available-dispatch-instances` is smaller than `initial_instances` (especially when `--enable-scaling` is not set), otherwise there would be no instances for decoding.
\ No newline at end of file
+Llumnix offers two methods for deploying P-D disaggregation service. You first need to start Ray in cluster, then follow the [Quickstart](./Quickstart.md) to configure environment variables HEAD_NODE_IP and HEAD_NODE for Llumnix.
+
+If Llumnix parameters are specified using a configuration file, ensure that `SERVER.LAUNCH_RAY_CLUSTER` in the config file is set to `False` to avoid disrupting the Ray setup.
+
+### Migrating from Existing Deployments
+
+Below are examples of how to launch prefill and decode instance:
+
+```bash
+# launch prefill instance
+python -m llumnix.entrypoints.vllm.api_server --host `hostname -i` --port $PORT --model $MODEL_PATH --worker-use-ray --trust-remote-code --max-model-len 4096 --config-file configs/vllm.yml --enable-pd-disagg --instance-type prefill --initial-instances 1
+
+# launch decode instance
+python -m llumnix.entrypoints.vllm.api_server --host `hostname -i` --port $PORT --model $MODEL_PATH --worker-use-ray --trust-remote-code --max-model-len 4096 --config-file configs/vllm.yml --enable-pd-disagg --instance-type decode --initial-instances 1
+```
+
+In this deployment model, `--instance-type` must be explicitly specified for every launch command, and `--initial-instances` will control how many instances of the corresponding type are launched.
+
+Note: It is the user's responsibility to handle failover in this kind of deployment model. If only prefill instances exist and no decode instances are available, requests will hang in the prefill instances.
+
+### Centralized Launch
+
+Centralized launch assumes that users have already launch a Ray cluter. Upon running the serve module, Llumnix will automatically connect to the existing Ray cluster, start the Llumnix components, and deploy multiple servers and instances. The `serve` entrypoint is easy to submit through the Ray job. Users can submit the Llumnix launch job directly via the Ray job submission API using the serve entrypoint. Below is example of centralized launch command:
```

Comment:
just a copy-paste from centralized launch? i remember that there are some differences between pdd in local launch and pdd in global launch.